beacon_node/network/src/sync/network_context.rs

```diff
@@ -496,6 +527,100 @@ impl<T: BeaconChainTypes> SyncNetworkContext<T> {
         Ok(())
     }
 
+    pub fn custody_lookup_request(
+        &mut self,
+        lookup_id: SingleLookupId,
+        block_root: Hash256,
+        downloaded_block_expected_data: Option<usize>,
+    ) -> Result<bool, &'static str> {
+        // Check if we are into peerdas
+        if !self
+            .chain
+            .data_availability_checker
+            .data_columns_required_for_epoch(
+                // TODO(das): use the block's slot
+                self.chain
+                    .slot_clock
+                    .now_or_genesis()
+                    .ok_or("clock not available")?
+                    .epoch(T::EthSpec::slots_per_epoch()),
+            )
+        {
+            return Ok(false);
+        }
+
+        let expects_data = downloaded_block_expected_data
+            .or_else(|| {
+                self.chain
+                    .data_availability_checker
+                    .num_expected_blobs(&block_root)
+            })
+            .map(|n| n > 0)
+            // If we don't know about the block being requested, assume block has data
+            .unwrap_or(true);
+
+        // No data required for this block
+        if !expects_data {
+            return Ok(false);
+        }
+
+        let custody_indexes_imported = self
+            .chain
+            .data_availability_checker
+            .imported_custody_column_indexes(&block_root)
+            .unwrap_or_default();
+
+        // TODO(das): figure out how to pass block.slot if we end up doing rotation
+        let block_epoch = Epoch::new(0);
```

Comment:
Feels like we should just remove the epoch from the `custody_columns` function until we need it - adding this dummy value could be a source of confusion / bugs in the future, in case things change or if we log this epoch.